packages/firestore/test/integration/api/query.test.ts

```diff
@@ -435,10 +435,6 @@ apiDescribe('Queries', persistence => {
 
   it('can query while reconnecting to network', () => {
     return withTestCollection(persistence, /* docs= */ {}, coll => {
-      // TODO(mikelehen): Find better way to expose this to tests.
-      // tslint:disable-next-line:no-any enableNetwork isn't exposed via d.ts
-      const firestoreInternal = coll.firestore.INTERNAL as any;
```

Comment:
There's another one of these in the next test (which you picked up when you rebased on firestore-api-changes... sorry :-))

Reply:
Done